schemas/follow_up.json

```diff
@@ -288,7 +288,9 @@
       "description": "The code to represent the stage group of the tumour, as assigned by the reporting posttherapy_tumour_staging_system, that indicates the overall prognostic tumour stage (ie. Stage I, Stage II, Stage III etc.) after treatment for patients receiving systemic and/or radiation therapy alone or as a component of their initial treatment, or as neoadjuvant therapy before planned surgery.",
       "valueType": "string",
       "meta": {
-        "dependsOn": "follow_up.post_therapy_tumour_staging_system"
+        "dependsOn": "follow_up.post_therapy_tumour_staging_system",
+        "notes": "This field value is dependent on the selected postetherapy_tumour_staging_system."
```

Comment:
post-therapy? or posttherapy? I think there's an extra 'e' :)

Reply:
copy-pasta'd.  fixed all of them.